sdk/postgresql/arm-postgresql-flexible/tsconfig.browser.config.json

```diff
@@ -1,3 +1 @@
-{
-  "extends": ["./tsconfig.test.json", "../../../tsconfig.browser.base.json"]
-}
+{ "extends": ["./tsconfig.test.json", "../../../tsconfig.browser.base.json"] }
```

Comment:
why format doesn't work here?

Reply:
this is generated from codegen, it doesn't format this code:
```
 project.createSourceFile("tsconfig.browser.config.json", JSON.stringify(highLevelTsBrowserConfig), {
        overwrite: true
    });
```
not
```
project.createSourceFile("tsconfig.browser.config.json", JSON.stringify(highLevelTsBrowserConfig, null, 2), {
    overwrite: true
  });
```